assets/src/components/button/style.scss

```diff
@@ -0,0 +1,53 @@
+/**
+ * Button styles,
+ */
+
+.muriel-button {
+	text-align: center;
+	font-size: 14px;
+	line-height: 18px;
+	font-weight: 500;
+	color: $muriel-gray-700;
+	text-shadow: none;
+
+	&.is-button {
+		display: inline-block;
+		background-color: $muriel-white;
+		border-color: #CCCCCC;
+		border-radius: 3px;
+		cursor: pointer;
```

Comment:
Tiny nit: don't think you need `cursor: pointer` here since you get it for free from `.components-button`

Reply:
Copy pasta!